rmgpy/solver/liquid.pyx

```diff
@@ -348,14 +348,17 @@ cdef class LiquidReactor(ReactionSystem):
                         if third >= num_core_species: edge_species_rates[third - num_core_species] += reaction_rate
 
         for j in range(inet.shape[0]):
-            k = knet[j]
-            if inet[j, 1] == -1:  # only one reactant
-                reaction_rate = k * C[inet[j, 0]]
-            elif inet[j, 2] == -1:  # only two reactants
-                reaction_rate = k * C[inet[j, 0]] * C[inet[j, 1]]
-            else:  # three reactants
-                reaction_rate = k * C[inet[j, 0]] * C[inet[j, 1]] * C[inet[j, 2]]
-            network_leak_rates[j] = reaction_rate
+            if inet[j, 0] != -1: #all source species are in the core
+                k = knet[j]
+                if inet[j, 1] == -1:  # only one reactant
+                    reaction_rate = k * C[inet[j, 0]]
+                elif inet[j, 2] == -1:  # only two reactants
+                    reaction_rate = k * C[inet[j, 0]] * C[inet[j, 1]]
+                else:  # three reactants
+                    reaction_rate = k * C[inet[j, 0]] * C[inet[j, 1]] * C[inet[j, 2]]
+                    network_leak_rates[j] = reaction_rate
+            else:
+                network_leak_rates[j] = 0.0
```

Comment:
what's the meaning of a (pdep) network in the context of a liquid reactor?

Reply:
It shouldn't exist in a liquid reactor, but the code must have been copied over when making the liquid reactor file.

Reply:
do we want/need to keep it?

Reply:
We certainly don't need or want it, but it's not the only leftover from pdep, and these things may have been left in for good reason. I think that's separate from this PR and I'm not sure how worth exploring it is if we're planning to replace these with RMS.